services/src/atdd-staging/src/main/java/org/openkilda/atdd/staging/service/traffexam/FlowNotApplicableException.java

```diff
@@ -0,0 +1,8 @@
+package org.openkilda.atdd.staging.service.traffexam;
+
+public class FlowNotApplicableException extends Exception {
```

Comment:
When adding a new exception, it's good to describe why/how it is used ... ie which class required this.